Add section about BI dashboards & Metabase (#531)
objectiv/objectiv.io: docs/docs/home/quickstart-guide.md

> * `objectiv_collector` An **Objectiv Collector** to validate & store event data from the tracker
> * `objectiv_postgres` A PostgreSQL database, pre-filled with anonymized demo user data
> * `objectiv_notebook` A Jupyter Notebook environment to demonstrate the **open model hub** and **Objectiv Bach**  
> * `objectiv_metabase` Open-source BI tool Metabase, to view how you can quickly build dashboards using Objectiv  

Reviewer: ```suggestion
* `objectiv_collector`: An **Objectiv Collector backend** to validate & store event data from the tracker.
* `objectiv_postgres`: A PostgreSQL database, pre-filled with anonymized demo user data.
* `objectiv_notebook`: A Jupyter Notebook environment to demonstrate the **open model hub** and **Objectiv Bach** .
* `objectiv_metabase`: Open-source BI-tool Metabase, to see how you can quickly build dashboards with Objectiv.
```

```
For detailed modeling instructions, check out our [modeling docs](/modeling/index.mdx).

### Creating BI dashboards

> For any DataFrame or Series that you create using Objectiv in your notebook, you can get the SQL for this exact analysis on your full data warehouse. This then can be used to quickly build dashboards in any SQL supporting BI tool.

Reviewer: ```suggestion
For any DataFrame or Series that you create with Objectiv in your notebook, you can get the SQL that runs directly on your full data warehouse. This can then be used to quickly build dashboards in any SQL supporting BI tool.
```

Steps to take:
1. Generate the SQL for any [DataFrame](/modeling/bach/api-reference/DataFrame/view_sql/) or [Series](/modeling/bach/api-reference/Series/view_sql/) in your notebook. 
2. Open your BI tool that has access to the same data warehouse and run the query. In the demo, you can [run Metabase locally](http://localhost:3000) and then create a new SQL query. For the Metabase demo, use these credentials:

Username: 
```bash 
    demo@objectiv.io
```

Password: 
```bash 
    metabase1 
```

> 3. Copy-paste the generated SQL from your notebook in Metabase, execute this and choose a visualisation for a dashboard view.

Reviewer: ```suggestion
3. Copy-paste the generated SQL from your notebook into Metabase, execute it, and choose a visualization for a dashboard view.
```
